org.endlessos.Key.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: org.endlessos.Key
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: /app/bin/kolibri-gnome
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --system-talk-name=org.endlessos.Key.Daemon
+  - --env=KOLIBRI_HOME=~/.var/app/org.endlessos.Key/data/kolibri
+  - --env=KOLIBRI_HTTP_PORT=0
+  - --env=PYTHONPATH=/app/kolibri-plugins/lib/python
+
+add-extensions:
+  org.learningequality.Kolibri.Content:
+    version: '1.0'
+    directory: share/kolibri-content
+    subdirectories: true
+    no-autodownload: true
+
+  org.learningequality.Kolibri.Plugin:
+    version: '1.0'
+    directory: kolibri-plugins
+    merge-dirs: lib/python
+    subdirectories: true
+    no-autodownload: true
+
+build-options:
+  env:
+    KOLIBRI_MODULE_PATH: /app/lib/python3.10/site-packages/kolibri
+  test-args:
+    - --socket=fallback-x11
+    - --socket=wayland
+
+modules:
+  - modules/iproute2.json
+  - modules/python3-markdown.json
+  - modules/python3-kolibri.json
+  - modules/python3-kolibri-pytz.json
+  - modules/python3-kolibri-app-desktop-xdg-plugin.json
+  - modules/python3-kolibri-desktop-auth-plugin.json
+  - modules/kolibri-home-template.json
+  - modules/python3-setproctitle.json
+  - modules/python3-virtualenv-api.json
+  - modules/kolibri-content-dir.json
+  - modules/kolibri-plugins-dir.json
+
+  - name: kolibri-gnome
+    buildsystem: meson
+    builddir: true
+    run-tests: true
+    config-opts:
+      - -Dkolibri_home_template_dir=/app/share/kolibri-home-template
+      - -Dprofile=default
+    sources:
+      - type: git
+        url: https://github.com/endlessm/endless-key-flatpak.git
+        tag: endlesskey_1.0.0
+        commit: 927b8022cc25facc11fe38ccd20ab2c04b49da1b
+
+  - name: python3-klobri-cleanup
+    buildsystem: simple
+    build-commands:
+      # GNOME 44 runtime has Python 3.10
+      - find ${KOLIBRI_MODULE_PATH}/dist/cext -type d -name 'cp*' -not -path ${KOLIBRI_MODULE_PATH}/dist/cext/cp310 -exec rm -rf '{}' '+'
+      - rm -rf ${KOLIBRI_MODULE_PATH}/dist/cext/*/Windows
+      - find ${KOLIBRI_MODULE_PATH} -type d -name 'test*' -not -path ${KOLIBRI_MODULE_PATH}/dist/django/test -exec rm -rf '{}' '+'
+      - find /app/lib/python3.10 /app/share/kolibri-home-template -name '*.js.map' -exec rm '{}' '+'
+
+  - name: kolibri-tools
+    buildsystem: simple
+    build-commands:
+      - install -m 755 -t ${FLATPAK_DEST}/bin kolibri-listcontent.py
+    sources:
+      - type: dir
+        path: tools
```

Comment:
I think `kolibri-listcontent.py` is still useful for Endless Key, but I am reminded _now_ is a good time to quit having this in the flatpak repo. Wouldn't really help to move it to our kolibri-gnome fork. Maybe we can propose it upstream as a new management command? I don't think we actually _did_ at any point, and it makes more sense now that the other content manifest stuff is in place. (Although it needs a rewrite to reuse code that is now actually upstream).

Reply:
Yes!  I was stuck how to deal **kolibri-listcontent.py**.  Moving it to our kolibri-gnome fork is a good idea.

Reply:
Have the PR [Install kolibri-listcontent.py](https://github.com/endlessm/endless-key-flatpak/pull/30)